src/components/Main.js

```diff
@@ -44,6 +45,9 @@ export default function Main({ onHandleSubmit }) {
     setArrivalRunway(runway);
     setArrivalLocation({ lat, lon });
   }
+  function selectAircraft(airplane) {
+    setAircraft(airplane);
+  }
```

Comment:
do you really need a function for that?

Reply:
Thanks for the feedback Felix, all done!